docs/app/components/page-footer/page-footer.component.less

```diff
@@ -40,9 +40,20 @@
         margin-bottom: 0;
     }
 
+    .page-footer-brand-logo-container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
     .page-footer-brand-logo {
-        display: block;
-        margin: 0 auto;
+        margin-right: 5px
+    }
+
+    .page-footer-brand-logo-title {
+        color: #ffffff;
```

Comment:
Please use the variables as much as possible - it will make it easier to override for the Micro Focus site (which will be black rather than white here, for example).